src/Concerns/HasContent.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+namespace Plank\Contentable\Concerns;
+
+use Illuminate\Database\Eloquent\Collection as EloquentCollection;
```

Comment:
Paired with other comment.

```suggestion
use Illuminate\Database\Eloquent\Collection as EloquentCollection;
use Illuminate\Database\Eloquent\Model;
```